docs/en/themes/bugzilla/static/bugzilla.css_t

```diff
@@ -0,0 +1,141 @@
+/*
+ * bugzilla.css_t
+ * ~~~~~~~~~~~~~~
+ *
+ * Sphinx stylesheet -- bugzilla theme.
+ *
+ * :copyright: Copyright 2007-2022 by the Sphinx team, see AUTHORS.
+ * :license: BSD, see LICENSE for details.
+ *
+ */
+
+@import url("classic.css");
+
+/* -- page layout ----------------------------------------------------------- */
+
+@font-face {
+    font-family: 'FiraGO Light';
+    font-style: normal;
+    font-weight: normal;
+    src: url('fonts/Roman/FiraGO-Light.eot');
+    src: url('fonts/Roman/FiraGO-Light.eot?#iefix') format('embedded-opentype'),
+         url('fonts/Roman/FiraGO-Light.woff2') format('woff2'), url('fonts/Roman/FiraGO-Light.woff') format('woff');
+}
+
+@font-face {
+    font-family: 'FiraGO Regular';
+    font-style: normal;
+    font-weight: normal;
+    src: url('fonts/Roman/FiraGO-Regular.eot');
+    src: url('fonts/Roman/FiraGO-Regular.eot?#iefix') format('embedded-opentype'),
+         url('fonts/Roman/FiraGO-Regular.woff2') format('woff2'), url('fonts/Roman/FiraGO-Regular.woff') format('woff');
+}
+
+@font-face {
+    font-family: 'FiraMono Bold';
+    font-style: normal;
+    font-weight: normal;
+    src: url('fonts/Mono/FiraMono-Bold.eot');
+    src: url('fonts/Mono/FiraMono-Bold.eot?#iefix') format('embedded-opentype'),
+         url('fonts/Mono/FiraMono-Bold.woff2') format('woff2'), url('fonts/Mono/FiraMono-Bold.woff') format('woff');
+}
+
+@font-face {
+    font-family: 'FiraMono Regular';
+    font-style: normal;
+    font-weight: normal;
+    src: url('fonts/Mono/FiraMono-Regular.eot');
+    src: url('fonts/Mono/FiraMono-Regular.eot?#iefix') format('embedded-opentype'),
+         url('fonts/Mono/FiraMono-Regular.woff2') format('woff2'), url('fonts/Mono/FiraMono-Regular.woff') format('woff');
+}
+
+div.document {
+    min-height: 95vh;
```

Comment:
This is an ugly hack, please make the page use flex grow or something that always pushes the footer to the bottom, without doing this, or do not do this at all

Reply:
True, will remove it!